prowler/providers/gcp/gcp_provider.py

```diff
@@ -212,7 +226,9 @@ def get_output_mapping(self):
         }
 
     @staticmethod
-    def setup_session(credentials_file: str, service_account: str) -> tuple:
+    def setup_session(
+        credentials_file: str, service_account: str, gcp_credentials: dict = None
```

Comment:
We need to make the arguments optional.

Reply:
The only way to access this method it's via `__init__` or `test_connection` and in both of them this arguments have a default value